build.gradle

```diff
@@ -1,55 +1,55 @@
 plugins {
-    id 'java'
-    id 'maven-publish'
+	id 'java-library'
+	id 'maven-publish'
 }
 
 group 'net.fabricmc'
 archivesBaseName = 'lorenz-tiny'
-version '2.0.0'
+version '2.1.0'
 
 def ENV = System.getenv()
 version = version + "+" + (ENV.BUILD_NUMBER ? ("build." + ENV.BUILD_NUMBER) : "local")
 
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
-repositories {
-    mavenCentral()
-    maven {
-        name = 'Fabric'
-        url = 'https://maven.fabricmc.net/'
-    }
+java {
+	withSourcesJar()
+	withJavadocJar()
 }
 
-dependencies {
-    implementation 'net.fabricmc:tiny-mappings-parser:0.2.1.13'
-    implementation 'org.cadixdev:lorenz:0.5.2'
+repositories {
+	mavenCentral()
+	maven {
+		name = 'Fabric'
+		url = 'https://maven.fabricmc.net/'
+	}
 }
 
-task sourcesJar(type: Jar) {
-    classifier 'sources'
-    from sourceSets.main.allJava
+dependencies {
+	implementation 'org.cadixdev:lorenz:0.5.6'
```

Comment:
Isn't Lorenz also an `api` dependency since lorenz-tiny exposes it in method signatures?

Reply:
Yeah you're right, will update.